src/cmd/flux-job.c

```diff
@@ -1588,7 +1588,7 @@ struct attach_ctx {
     flux_jobid_t id;
```

Comment:
commit message for  095ed41af417cf78801bc611404b3c7f4b23fb5a:
s/prodided/provided/

Reply:
Thanks for catching that! I've fixed that and force pushed, and will set MWP.